.circleci/config.yml

```diff
@@ -35,8 +36,8 @@ jobs:
       - persist_to_workspace:
           root: .
           paths:
-            - ./venv
-            - ./node_modules
+            - venv
+            - node_modules
```

Comment:
Are these necessary changes, or side-effects from lots of small tweaks trying to get CI to pass?

Reply:
The latter.